src/gui/src/Widgets/Rotary.cpp

```diff
@@ -224,33 +234,44 @@ void Rotary::wheelEvent ( QWheelEvent *ev )
 {
 	ev->accept();
 
-	float stepfactor = 5.0; // course adjustment
+	float stepfactor = 5.0; // coarse adjustment
 	float delta = 1.0;
 
 	// Control Modifier = fine adjustment
 	if (ev->modifiers() == Qt::ControlModifier) {
 		stepfactor = 1.0;
 	}
 	if ( !m_bUseIntSteps ) {
-		float fRange = fabs( m_fMax ) + fabs( m_fMin );
+		float fRange = m_fMax - m_fMin;
 		delta = fRange / 100.0;
 	}
 	if ( ev->angleDelta().y() < 0 ) {
-		delta = delta * -1.0;
+		delta *= -1.;
 	}
 	setValue( getValue() + (delta * stepfactor) );
 	emit valueChanged(this);
+	
+	if ( m_bShowValueToolTip ) {
+		char tmp[20];
+		sprintf( tmp, "%#.2f", m_fValue );
+		
+		// Problem: ValueToolTip remains visible after wheel event
+		//m_pValueToolTip->showTip( mapToGlobal( QPoint( -38, 1 ) ), QString( tmp ) );
+		
+		// useful but graphically incoeherent
+		QToolTip::showText( ev->globalPos(), QString( "%1" ).arg( m_fValue, 0, 'f', 2 ) , this );
```

Comment:
Personally the graphical incoherence is not a problem for me, nor a priority at this point ( and I guess even for you ;) )
But yes I agree, I just prefer now to spend time in bugs and the new features like the float ticks and tuplets...
That pull req. was just for the sake of "reading the numbers", waiting for a new GUI eventually.

Reply:
Of course if you already know how to use Qt it is fine to include the change here

Reply:
> Personally the graphical incoherence is not a problem for me, nor a priority at this point ( and I guess even for you ;) )

Well, it is not on my todo list and could easily wait. But the GUI really needs an update (in the graphics -> port to SVG, the components -> using .ui files to represent e.g. the Instrument Editor instead of a solid bitmap with no internationalization, and the general UX -> like keyboard interaction, click&drag). And the fundamental widgets are a good place to start. A year ago I wouldn't have minded. But change of Colin to the Song and Pattern Editor showed me that tweaks in the UX can improve an old-fashioned app soo much it actually becomes pleasant to work with.

I think I will have another look at the tooltips to make them consistent for all widgets, try to make their look integrate better into the overall design of Hydrogen, and replace both the Rotary and Knob images by SVG.

So, if you want to spend time with other issues, I would suggest we merge this PR and I take over in a different one.

Reply:
> So, if you want to spend time with other issues, I would suggest we merge this PR and I take over in a different one.

Perfect!